From fc7271aeff296034ef51ca039b2bea3de30053d4 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Mon, 8 Oct 2018 19:48:56 +0800
Subject: [PATCH] Revert "arm64: dts: rockchip: fix reg length for cru on
 RK1808 SoCs"

This reverts commit 827031a1cabb32278405455db10777dfcdebbbae.

PMU_CRU offset is 0x4000

Change-Id: Ia535a255030b530949bc205414dca86ffd7ff6b6
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index b811e82ad568..00eb20c03c3d 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -301,7 +301,7 @@
 
 	cru: clock-controller@ff350000 {
 		compatible = "rockchip,rk1808-cru";
-		reg = <0x0 0xff350000 0x0 0xd000>;
+		reg = <0x0 0xff350000 0x0 0x5000>;
 		rockchip,grf = <&grf>;
 		#clock-cells = <1>;
 		#reset-cells = <1>;
-- 
2.35.3

